projects/sd-jwt-js.md

```diff
@@ -0,0 +1,80 @@
+# Project Name
+
+SD-JWT Javascript Reference Implementation
+
+# Project Description
+
+This is the reference implmentation of [IETF SD-JWT specification](https://datatracker.ietf.org/doc/draft-ietf-oauth-selective-disclosure-jwt/) in Javascript.
+
+Hopae, a founding member of OpenWallet Foundation, is building wallet module in Javascript and need this project as a core component.
+
+# Alignment with the OpenWallet Foundation Mission
+
+This library is for the Selective Disclosure capability which is one of the important building blocks of digital identity wallets.
+
+# Current Code of Conduct
+
+none
```

Comment:
Is it fine if we use the LFEU code of conduct (found at https://linuxfoundation.eu/en/policies)